pkg/auth/api/api.go

```diff
@@ -295,75 +293,3 @@ func (s *authService) maybeCheckEmail(ctx context.Context, email string) error {
 	)
 	return localizedError(statusAccessDeniedEmail, locale.JaJP)
 }
-
```

Comment:
I am curious as to why we need to remove this code.

Reply:
`checkAdminRole` and `checkRole` in `pkg/auth/api/api.go` are not used, so golangci-lint pointed out it.

These same codes exist and used in many files like `pkg/account/api/admin_account.go` and `pkg/environment/api/api.go`.

Reply:
I see, thank you!